### ISSUE TITLE
Slack channel for non-contributors

### ISSUE THREAD

**Issue by worldofgeese** [gopasspw/gopass#1761]
We just had someone leave our company and considered it might be a good idea to switch to the age backend to secure our secrets from exfiltration. I thought oh maybe I should discuss the age backend with some like-minded gopass people but it looks like the Slack workspace is for contributors only. 

Is it possible to open up the workspace to non-contributors perhaps siloed to their own channel so we don't cause a headache for the developers?

**Comment by dominikschulz** [comment]
The Slack workspace is intended for everyone who's interested. Especially early adopters.

Maybe we can make it more accessible?

**Comment by worldofgeese** [comment]
Perhaps listing a shared invite URL?

**Comment by dominikschulz** [comment]
Yes, that would be good. But I'm not sure what the best way to implement that would be.
I have to admit that this is one of the down sides of Slack :/

**Comment by dominikschulz** [comment]
I've update the README to include that link.
But I'll need someone to test it. I don't want to go through the whole signup workflow only to test it.
